### ISSUE TITLE
Automate removing things from NOR dashboard after they're accepted in OBO Foundry

### ISSUE THREAD

**Issue by cthoyt** [OBOFoundry/obo-nor.github.io#22]
Would it make sense to automate removing things from the NOR dashboard after they're accepted in OBO Foundry?

The steps that could be taken in a script in a GitHub Actions workflow running once nightly or on request:

1. Get all ontologies' prefixes from https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/registry/ontologies.yml
2. Filter all entries in https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml that have a prefix that's in the OBO Foundry already
3. Commit the `dashboard-config.yml` back to the repo if there are any changes

**Comment by matentzn** [comment]
Yeah, good idea. However, we will have a curator appointed to this task anyways soon, so I wouldn't go nuts with personal time resources on this one.

**Comment by cthoyt** [comment]
I estimate it will take 10 minutes to get this working. I'm sure the person doing it manually will take much more effort than this to train and spend on it :p

**Comment by cthoyt** [comment]
I'll do this over the weekend. When you assign someone to this role, we can check see if they can understand it and where it needs more docs.
